README.md

```diff
@@ -104,70 +96,13 @@ ansible-playbook -e rm_dest=~/github/rm_example \
 │                   ├── __init__.py
 │                   └── utils.py
 ├── README.md
-├── roles
-
- 
-```
-**Role directory layout**
+├── tests
 
-- `network_os`: myos
-- `resource`: interfaces
-
-```
-ansible-playbook -e rm_dest=~/github/rm_example/roles/my_role \
-                 -e structure=role \
-                 -e model=models/myos/interfaces/myos_interfaces.yml \
-                 site.yml
-```
-
-```
-roles
-└── my_role
-    ├── library
-    │   ├── __init__.py
-    │   ├── myos_facts.py
-    │   └── myos_interfaces.py
-    ├── LICENSE.txt
-    ├── module_utils
-    │   ├── __init__.py
-    │   └── network
-    │       ├── __init__.py
-    │       └── myos
-    │           ├── argspec
-    │           │   ├── facts
-    │           │   │   ├── facts.py
-    │           │   │   └── __init__.py
-    │           │   ├── __init__.py
-    │           │   └── interfaces
-    │           │       ├── __init__.py
-    │           │       └── interfaces.py
-    │           ├── config
-    │           │   ├── base.py
-    │           │   ├── __init__.py
-    │           │   └── interfaces
-    │           │       ├── __init__.py
-    │           │       └── interfaces.py
-    │           ├── facts
-    │           │   ├── base.py
-    │           │   ├── facts.py
-    │           │   ├── __init__.py
-    │           │   └── interfaces
-    │           │       ├── __init__.py
-    │           │       └── interfaces.py
-    │           ├── __init__.py
-    │           └── utils
-    │               ├── __init__.py
-    │               └── utils.py
-    └── README.md
-```
 
 **Using the collection layout**
 
 ```
-git clone git@github.com:ansible/ansible.git
-cd ansible
-git fetch origin pull/52194/head:collection_test
-git checkout collection_test
+pip install ansible-base==2.10.0b1 --user
```

Comment:
Can you add a sentence on what this is doing? I'm not clear what installing ansible-base has to do with 'using the collection layout'.

Reply:
I believe the original intention was to show how to use collections in a playbook. But now that we have comprehensive docs that cover this topic, I decided to add a link to those, instead of duplicating it here.